src/pages/components/image-with-caption.mdx

```diff
@@ -29,6 +30,16 @@ The Image with caption supports images at 1x1, 2x1, 4x3, and 16x9 aspect ratios.
 </Column>
 </Row>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline (caption) | Short copy | Yes | 1 | XXS (40 / 55)|
+| Description | Long copy | No | 1 | L (150 / 200)|
+| Image | Media | Yes | 1 | - |
+| [Lightbox media viewer](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/lightbox-media-viewer) | Component | No | 1 | If enabled, the user can view the image and description in the lightbox modal by selecting the image. |
+
+To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank">link</a>.
```

Comment:
I ***think*** there is a linting error here if you have a `_blank` with no `rel="noreferrer"`

Reply:
You are right on sir! Great catch. i'll update all the links to have that.